WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/Subscription/SubscriptionExpiryViewModelTests.swift

```diff
@@ -0,0 +1,129 @@
+import XCTest
+import Yosemite
+@testable import WooCommerce
+import WooFoundation
+
+final class SubscriptionExpiryViewModelTests: XCTestCase {
+    func test_selectedLength_has_correct_value() {
+        // Given
+        let subscription = ProductSubscription.fake().copy(length: "0")
+        let viewModel = SubscriptionExpiryViewModel(subscription: subscription) { _, _ in }
+
+        // Then
+        XCTAssertEqual(viewModel.selectedLength.title, "Never expire")
```

Comment:
np, possibly same situation as in this comment: https://github.com/woocommerce/woocommerce-ios/pull/11173#discussion_r1396615549

Reply:
Good catch. Also fixed in 887a46459098d564bd418e024a73f3ad96d087d7